account_easy_reconcile/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Easy Reconcile
+==============
+
+This is a shared work between Akretion and Camptocamp
+in order to provide:
+ - reconciliation facilities for big volume of transactions
+ - setup different profiles of reconciliation by account
+ - each profile can use many methods of reconciliation
+ - this module is also a base to create others
+   reconciliation methods which can plug in the profiles
+ - a profile a reconciliation can be run manually
+   or by a cron
+ - monitoring of reconciliation runs with an history
+   which keep track of the reconciled Journal items
+
+2 simple reconciliation methods are integrated
+in this module, the simple reconciliations works
+on 2 lines (1 debit / 1 credit) and do not allow
+partial reconcilation, they also match on 1 key,
+partner or Journal item name.
+
+You may be interested to install also the
+``account_advanced_reconciliation`` module.
+This latter add more complex reconciliations,
+allows multiple lines and partial.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/bank-statement-reconcile/issues/new?body=module:%20account_easy_reconcile%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
```

Comment:
Here the history is missing:
https://github.com/OCA/bank-statement-reconcile/commits/7.0/account_easy_reconcile

In order of contrib:

@sebastienbeau
@guewen 
@vrenaville 
@gurneyalex 
@jgrandguillaume 
@nbessi
@pedrobaeza 
@mdietrichc2c 
@lepistone 
@ecino 
@yvaucher
@rschnapka
@florian-dacosta 
@laetitia-gangloff 

See git log for emails